containers/daap-delete-table-for-data-product/src/var/task/delete_table.py

```diff
@@ -0,0 +1,48 @@
+import json
+import os
+from http import HTTPMethod
+
+import boto3
+from data_platform_api_responses import (
+    response_status_200,
+    response_status_400,
+    response_status_403,
+)
+from data_platform_logging import DataPlatformLogger, s3_security_opts
+from data_platform_paths import DataProductConfig, get_latest_version, get_new_version
+from data_product_metadata import DataProductMetadata, DataProductSchema
+
+s3_client = boto3.client("s3")
+glue_client = boto3.client("glue")
+athena_client = boto3.client("athena")
+
+
+def handler(event, context):
+    data_product_name = event["pathParameters"].get("data-product-name")
+    table_name = event["pathParameters"].get("table-name")
+    request_body = json.loads(event.get("body"))
+    http_method = event.get("httpMethod")
+    logger = DataPlatformLogger(
+        extra={
+            "image_version": os.getenv("VERSION", "unknown"),
+            "base_image_version": os.getenv("BASE_VERSION", "unknown"),
+            "data_product_name": data_product_name,
+            "table_name": table_name,
+        }
+    )
+
+    logger.info(f"event: {event}")
+
+    if not http_method == HTTPMethod.POST:
```

Comment:
I'm not sure if we need to worry about checking the method here - I would expect this to be enforced at the API gateway?